tfexec/options.go

```diff
@@ -243,6 +243,15 @@ func GraphPlan(file string) *GraphPlanOption {
 	return &GraphPlanOption{file}
 }
 
+type UseJSONNumber struct {
```

Comment:
```suggestion
type UseJSONNumberOption struct {
```

Naming convention for the struct.